MobileBuy/buy3/src/main/java/com/shopify/buy3/GraphClient.kt

```diff
@@ -42,7 +42,7 @@ import java.util.concurrent.TimeUnit
 
 private val DEFAULT_HTTP_CONNECTION_TIME_OUT_MS = TimeUnit.SECONDS.toMillis(10)
 private val DEFAULT_HTTP_READ_WRITE_TIME_OUT_MS = TimeUnit.SECONDS.toMillis(20)
-private val STOREFRONT_API_VERSION = "2020-01"
+private const val STOREFRONT_API_VERSION = Storefront.API_VERSION
```

Comment:
Nitpick: Do we have more than one reference to the api version in the Graph client? Instead of introducing a new constant, could we use `Storefront.API_VERSION` directly?

Reply:
Inlined the constant. We will do the `bundle` as a a part of the CI (Github Actions)